Gems/ScriptCanvasTesting/Code/Source/Framework/ScriptCanvasTestUtilities.cpp

```diff
@@ -57,7 +57,7 @@ namespace ScriptCanvasTests
     {
         using namespace ScriptCanvas;
 
-        AZ_TEST_START_TRACE_SUPPRESSION;
+  //      AZ_TEST_START_TRACE_SUPPRESSION;
```

Comment:
Is this code no longer throwing expected errors.
Because if not the commented trace suppression block should be removed.

Reply:
oh, this was a test and I didn't clean it up, will fix